src/manifest-creation.ts

```diff
@@ -73,7 +73,7 @@ export class ManifestCreation {
     return response.data.html_url
   }
 
-  private async updateEnv (env: any) {
+  public async updateEnv (env: any) {
```

Comment:
Hmm ts doesn't seem to like this: https://travis-ci.org/probot/probot/jobs/439276907#L481

Reply:
```
❯ yarn build
yarn run v1.10.1
$ rimraf lib && tsc -p tsconfig.json
src/manifest-creation.ts:76:45 - error TS2304: Cannot find name 'Env'.

76   public async updateEnv(env: any): Promise<Env> {
                                               ~~~


src/manifest-creation.ts:76:45 - error TS4055: Return type of public method from exported class has or is using private name 'Env'.

76   public async updateEnv(env: any): Promise<Env> {
                                               ~~~


error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

My understanding of this error after looking at the types with the `update-dotenv` module is that [`Env` is not exported](https://github.com/bkeepers/update-dotenv/blob/6fc786c9bd132dbcfb123eb440919849e607edfb/index.d.ts#L1-L3) (therefore, is a private interface). Since making this method public, I think TypeScript is now checking the return type more strictly, at least that's my guess.

I think the solution would be to:

* export the `Env` interface in `update-dotenv`:

```ts
export interface Env {
  [name: string]: string
}
```

* publish a new version of `update-dotenv` with the patched type definitions
* update this PR with the new version of `update-dotenv`

That would likely fix the issue, or we may additionally also need to update the import and type definition of the `updateEnv()` method, but I think that TypeScript would be able to infer properly with an exported `Env` interface:

```ts
import updateDotenv, { Env } from 'update-dotenv'

public async updateEnv (env: any): Promise<Env> { /* ... */ }
```